client/pages/form-pages/general-project-info.tsx

```diff
@@ -38,9 +38,15 @@ export const GeneralProjectInfo: React.FC = () => {
 
             <div className='flex flex-1 w-full justify-start'>
               <Radio
-                legend='What type of project is being applied for?'
+                legend={
+                  <>
+                    <p>What type of project is being applied for?</p>
+                    <p className='text-sm text-gray-500'>
+                      (Please read carefully and then select the one that best fits the project)
+                    </p>
```

Comment:
`<p>` funny enough, isn't a valid child of `<legend>` as they kind of overlap in purpose, I'd recommend `<span>` here

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend

Reply:
Imma bookmark it, very useful, thank you!